.github/draft_release_notes.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+
+# this script assumes the GITHUB_TOKEN and PREVIOUS_TAG environment variables have been set;
+# it produces a 'Changes.md' file as its final output;
+# the file 'last-300-prs-with-release-note.txt' that is produces is intermediate data; it is not
+# pruned for now to assist development of the release notes process (we are still curating all this)
+
```

Comment:
I am wondering whether it would be good to add checks for `GITHUB_TOKEN` and `PREVIOUS_TAG` just to avoid misleading error messages in case we mess up our GitHub Action YAML. Something like ...

```sh
if [ -z ${GITHUB_TOKEN+x} ]; then
  echo "Error: GITHUB_TOKEN is not set"
  ...
fi
```

Reply:
part of next push